src/components/Board/Board.js

```diff
@@ -31,28 +38,73 @@ const Board = ({ id }) => {
     setIsActiveForm(true);
   };
 
+  const handleOnDragEnd = (result) => {
+    const { destination, source, draggableId } = result;
+
+    if (!destination) {
+      return;
+    }
```

Comment:
What's your take on single line statements?
```suggestion
    if (!destination) return;
```